src/v3/test/integration/__snapshots__/authenticator-enroll-select-authenticator.test.tsx.snap

```diff
@@ -444,7 +445,7 @@ exports[`authenticator-enroll-select-authenticator renders form 1`] = `
                     <div
                       class="ods-4o5zYQ ods-2oc5SK ods-4XAbls ods-2NLKpK ods-1ll9ky ods-4IdjLs description"
                     >
-                      Enter a temporary code generated from an authenticator device.
```

Comment:
seems like two different messages

Reply:
Yep, this was another custom string we created in SIW Next, the udpated value is the one from SIW v2.

Reply:
Actually, correction, this was a fix, the IDP auth key was mistakenly pointing to the custom otp translation key, I fixed that issue here. You can see the same string below for the custom OTP.